tools/recetox_msfinder/macros.xml

```diff
@@ -0,0 +1,60 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.1.0</token>
+    <xml name="requirements">
+        <requirements>
+            <container type="docker">zargham12/msfinder:november</container>
+        </requirements>
+    </xml>
+    <xml name="creator">
+        <creator>
+            <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0002-6096-224X" />
+            <organization
+                url="https://www.recetox.muni.cz/"
+                email="GalaxyToolsDevelopmentandDeployment@space.muni.cz"
+                name="RECETOX MUNI" />
+        </creator>
+    </xml>
+    <xml name="input">
+        <param name="input_data" type="data" format="msp,mat">
+            <label>Input file to process</label>
+        </param>
+    </xml>
+    <xml name="parameter">
+        <param type="float" value="0.01" name="Ms2Tolerance" label="Ms2Tolerance" help="The mass tolerance for matching experimental- and reference fragments." />
+        <param name="IsTmsMeoxDerivative" type="boolean" checked="False" truevalue="True" falsevalue="False" label="IsTmsMeoxDerivative"
+            help="Check TMS-MeOX derivative to use EIMS database; otherwise, MSMS database is used." />
+        <param type="integer" value="1" name="RelativeAbundanceCutOff" label="RelativeAbundanceCutOff"
+            help="The product ions more than this parameter on the basis of base peak ion are utilized for the product ion matching." />
```

Comment:
I'm a bit confused by this help section: do fragments with abundance above or below this value get filtered out (not used)? 

As it seems from the description, those that are below this value are not used. In this case, however, the default value of `1` means no ions will be used for matching because no ion can have relative intensity (to a base peak) higher than `1`.

Reply:
```suggestion
        <param type="integer" value="1" name="RelativeAbundanceCutOff" label="RelativeAbundanceCutOff"
            help="Remove peaks with less then specified % relative intensity from the spectra." />
```